host/xtest/install_ta.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2017, Linaro Limited
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <compiler.h>
+#include <dirent.h>
+#include <err.h>
+#include <errno.h>
+#include <fnmatch.h>
+#include <inttypes.h>
+#include <pta_secstor_ta_mgmt.h>
```

Comment:
Build regression not addressed

Reply:
Don't just read/say that include too quickly :smile:

Reply:
Ha! Got it :wink: